src/mem/epoch/markable_atomic.rs

```diff
@@ -237,7 +264,7 @@ impl<T> MarkedAtomic<T> {
         unsafe { (Shared::from_raw(ptr), mark) }
     }
 
-    /// Perform a logical "or" on the current mark and the argument `mark` and set the new mark to
+    /// Perform a logical "OR" on the current mark and the argument `mark` and set the new mark to
```

Comment:
Lowercase "or" matches the standard library documentation. The standard library also doesn't use quotes, though.